subiquity/models/filesystem.py

```diff
@@ -553,6 +562,17 @@ class Partition(_Formattable):
     flag = attr.ib(default=None)
     preserve = attr.ib(default=False)
 
+    @property
+    def annotations(self):
+        r = super().annotations
+        if self.flag == "prep":
+            r.append("PReP")
+        elif self.flag == "boot":
+            r.append("ESP")
```

Comment:
I thought 'boot' flag is present on bios and prep partitions as well. Meaning that it's not ESP specific....

Reply:
no, flag is bios_grub or prep on those partitions. It's a string, not a list or a set...

Reply:
ah